com/laboon/WorldTest.java

```diff
@@ -0,0 +1,15 @@
+package com.laboon;
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+public class WorldTest {
+
+	//ensures the string built is not null
+	@Test
+	public void toStringNulltest() {
+		JavaLife JL = new JavaLife(10, 20, 25, 30);
```

Comment:
You should check World, not JavaLife.  Possible checks:

.toString when world is 0-length or small, checking side indices, create a world and check it against a known good string, many more..